src/gleam_stdlib.js

```diff
@@ -258,3 +258,67 @@ export function regex_scan(regex, string) {
   });
   return List.fromArray(matches);
 }
+
+export function new_map() {
+  return new Map();
+}
+
+export function map_size(map) {
+  return map.size;
+}
+
+export function map_to_list(map) {
+  const result = [];
+  map.forEach(([a, b]) => result.push([JSON.parse(a), b]))
+}
+
+export function map_from_list(list) {
+  return new Map(list.map(([a, b]) => [JSON.stringify(a), b]));
```

Comment:
Can we extract all instances of `JSON.stringify` into a function called `hashKey` or something? So in future we only have to update one place when we upgrade to a new hash function.